project/plugins.sbt

```diff
@@ -19,6 +19,6 @@ addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.5.5")
 
 addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.2.0")
 
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.5")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.7")
```

Comment:
There is Scala Native 0.4.8, but it could require:
```
libraryDependencySchemes ++= Seq(
  "org.scala-native" % "sbt-scala-native" % VersionScheme.Always
)
```

Reply:
Oh, let's try that

Reply:
Great! Waiting for a new release with impatience!